views/timeslot.tmpl

```diff
@@ -44,7 +44,7 @@
         alt="{{.Timeslot.Title}} Logo">
         <div class="row">
           <div class="col text-center">
-            <h6>Duration: {{.Timeslot.Duration}}</h6>
+            <span class="h6">Duration: </span><span>{{formatDuration .Timeslot.Duration}}</span>
```

Comment:
a span that pretends to be an h6? ...why?

Reply:
match the styling of a heading while making it inline. Its that or creating an inline class for headings